libbeat/idxmgmt/std.go

```diff
@@ -231,10 +231,16 @@ func (m *indexManager) load(forceTemplate, forcePolicy bool) error {
 
 	// install ilm policy
 	if withILM {
-		if err := m.ilm.EnsurePolicy(forcePolicy); err != nil {
+		policyCreated, err := m.ilm.EnsurePolicy(forcePolicy)
```

Comment:
wprobably `policyCreated` is not needed. We could receive from `EnsurePolicy(...)` into var `forceTemplate`

Reply:
we don't want to set forceTemplate to false if it was true already.